wikimedia/wikipedia-ios
# Updates for Xcode 12

### Fixes
https://phabricator.wikimedia.org/T259649

### Notes
* ~~As of Xcode 12 beta 4, there's a blocking iOS 14 simulator-only bug where articles take an excessively long time to load. The app gives the`webView` a `URLRequest` and there's a long pause (10+ seconds) before the `webView` makes the request. The issue persists if the custom scheme handler is removed and the request is made with `https` as the scheme. iOS < 14 simulators and devices work.~~ This is fixed in Xcode 12 beta 5
* Fixed an iPad crash with 1f7d04c
* Cleared view debugger warnings and fixed unresponsive explore feed cells in c00a5b2...97fa10b

### Test Steps
1. Build and run using the latest version of Xcode 12
2. Verify there are zero warnings
3. Verify articles, non-article pages, and editors load and work correctly
